### PR TITLE
Prep for beta release

### PR DESCRIPTION
**Pull request by rbarry82**
Rename all references from LMA->COS.

Make the libraries and logic more resilient to relation join
ordering and lifecycle events which may reset things.

Take the `nrpe_exporter` out of `.gitignore` after it snuck in.

Cap the maximum Jinja version so Charmcraft can pack it, and so
it aligns with the Grafana charm itself. Newer ones don't work
with old Pythons.

Update the tests

### CODE REVIEW

**Review by balbirthomas — APPROVED** [review]
Sanity checked all diffs.

**Review by balbirthomas — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# LMA Proxy charm
+# COS Proxy charm
 
-This Juju machine charm that provides a single integration point in the machine world with the Kubernetes-based [LMA bundle](https://charmhub.io/lma-light).
+This Juju machine charm that provides a single integration point in the machine world with the Kubernetes-based [COS bundle](https://charmhub.io/cos-light).
```
Comment (mmanciop) [inline README.md:3]: It’s “cos-lite”

**Review by mmanciop — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rbarry82) [reply README.md:3]: And "COS HA" eventually. Do the individual charms (especially the ones which will never be clustered) need to specify?

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Setup operator environment
+      uses: charmed-kubernetes/actions-operator@main
+      with:
+        juju-channel: latest/candidate # FIXME: Once 2.9.23 hits stable
+        provider: microk8s
+    - name: Run integration tests
+      run: tox -vve integration
```
Comment (simskij) [inline .github/workflows/ci.yaml:92]:
```suggestion
#  integration-test-microk8s:
#    name: Integration tests (microk8s)
#    runs-on: ubuntu-latest
#    steps:
#    - name: Checkout
#      uses: actions/checkout@v2
#    - name: Setup operator environment
#      uses: charmed-kubernetes/actions-operator@main
#      with:
#        juju-channel: latest/candidate # FIXME: Once 2.9.23 hits stable
#        provider: microk8s
#    - name: Run integration tests
#      run: tox -vve integration
```

Disable until we have any integration tests or the CI will continue to fail. 😅

**Review by simskij — DISMISSED** [review]

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (rbarry82) [reply .github/workflows/ci.yaml:92]: Actually will need to wait until they _also_ support machine charms whenever we have time for the "big" bundle framework

**Review by balbirthomas — APPROVED** [review]

**Review by sed-i — APPROVED** [review]

**Pull request merged**